cpp/include/cudf/strings/convert/convert_durations.hpp

```diff
@@ -115,15 +117,17 @@ std::unique_ptr<column> to_durations(
  *
  * @throw cudf::logic_error if `durations` column parameter is not a duration type.
  *
- * @param durations Duration values to convert.
+ * @param durations Duration values to convert
  * @param format The string specifying output format.
- *        Default format is ""%d days %H:%M:%S".
- * @param mr Device memory resource used to allocate the returned column's device memory.
- * @return New strings column with formatted durations.
+ *        Default format is ""%D days %H:%M:%S".
```

Comment:
Where do we stand on periods at the end of doxygen parameter descriptions? Remove them, like on line#120?
```suggestion
 *        Default format is ""%D days %H:%M:%S"
```

Reply:
https://github.com/rapidsai/cudf/pull/14247#discussion_r1349788189

Reply:
Thank you for that. Also, I wasn't aware of "Strunk & White".